libp2p/Host.cpp

```diff
@@ -91,15 +92,17 @@ Host::Host(string const& _clientVersion, KeyPair const& _alias, NetworkConfig co
                      // simultaneously
     m_tcp4Acceptor(m_ioService),
     m_runTimer(m_ioService),
-    m_alias(_alias),
+    m_alias(_secretAndENR.first),
```

Comment:
m_alias is a keypair and we need the pubkey to build the node's URL - since we're just passing in a secret now to the KeyPair ctor and secret is the private key, does this mean that the generated node URL will include the private key rather than the public key?

Reply:
No. Secret key and public key are different types (`Secret` and `Public`) and `KeyPair` constructor derives public key from secret key.

It was actually the same before, it happened in `Host::networkAlias` method

Reply:
Makes sense, thanks for the clarification!